lib/scan.ts

```diff
@@ -114,13 +115,25 @@ async function localArchiveAnalysis(
     // The target image becomes the base of the path, e.g. "archive.tar" for "/var/tmp/archive.tar"
     path.basename(archivePath);
 
+  let imageName: ImageName | undefined;
+  if (
+    (options.digests?.manifest || options.digests?.index) &&
```

Comment:
Why are these checks necessary - afaict both digests are optional in the `new ImageName` call, so I'm not sure why we need to ensure one is set :)

Reply:
Yeah that makes total sense. Thanks! :)